services/wallet/onramp/on_ramp_providers.go

```diff
@@ -1,145 +1,43 @@
+	)
+
+	ramps := []CryptoOnRamp{
+		{
+			Name:              "Ramp",
+			Description:       "Global crypto to fiat flow",
+			Fees:              "0.49% - 2.9%",
+			LogoURL:           logoRamp,
+			SiteURL:           "https://ramp.network/buy?hostApiKey=zrtf9u2uqebeyzcs37fu5857tktr3eg9w5tffove&swapAsset=DAI,ETH,USDC,USDT",
+			Hostname:          "ramp.network",
+			SupportedChainIDs: []uint64{walletCommon.EthereumMainnet, walletCommon.ArbitrumMainnet, walletCommon.OptimismMainnet},
+		},
+		{
+			Name:              "MoonPay",
+			Description:       "The new standard for fiat to crypto",
+			Fees:              "1% - 4.5%",
+			LogoURL:           logoMoonPay,
+			SiteURL:           "https://buy.moonpay.com/?apiKey=pk_live_YQC6CQPA5qqDu0unEwHJyAYQyeIqFGR",
+			Hostname:          "moonpay.com",
+			SupportedChainIDs: []uint64{walletCommon.EthereumMainnet, walletCommon.ArbitrumMainnet, walletCommon.OptimismMainnet},
+		},
+		{
+			Name:              "Latamex",
+			Description:       "Easily buy crypto in Argentina, Mexico, and Brazil",
+			Fees:              "1% - 1.7%",
+			LogoURL:           logoLatamex,
+			SiteURL:           "https://latamex.com/",
+			Hostname:          "latamex.com",
+			SupportedChainIDs: []uint64{walletCommon.EthereumMainnet},
+		},
```

Comment:
I don't see any providers with RecurrentSiteURL. Don't we still have any recurrent options?

Reply:
Oh, I see. We have a separate issue for that.

Reply:
Yeap! Mercuryo is coming as soon as we sign the deal! That will be our first recurrent-enabled provider, but I wanted to leave the API ready beforehand in case someone wants to start doing the client work.